### PR TITLE
Resolved size and multi-file upload issues.

### PR DESCRIPTION
**Pull request by palakrathi**

### CODE REVIEW

File: src/flowmanager.js [hunk 1]
```diff
@@ -109,20 +111,41 @@ export default class FlowManager extends EventEmitter {
...
+            data: file._data
+          });
+          this.emitUploader(internalEvents.UPLOAD_INITIATED, {
+            filename: file.name,
+            size: getHumanFileSize(fl.size),
+            progress: 0,
+            isComplete: false,
+            isError: false,
+            taskId: tempIds[index]
+          });
```
Comment (ophinjay) [inline src/flowmanager.js]: Repeated code.. Make it a utility function and use it both in the if condition and the loop in else

File: src/packet.js [hunk 2]
```diff
@@ -11,25 +11,24 @@ function getAllFileEntries(dataTransferItemList) {
     queue.push(dataTransferItemList[i].webkitGetAsEntry());
   }
 
-  const getEntries = () => {
+  const getEntries = queue => {
+    const promises = [];
     while (queue.length > 0) {
       let entry = queue.shift();
       if (entry.isFile) {
+        entry.file((fl) => console.log(fl));
```
Comment (ophinjay) [inline src/packet.js]: Remove console log

**Review by ophinjay — CHANGES_REQUESTED** [review]

File: src/flowmanager.js (hunk 1 above)
Comment (palakrathi) [reply src/flowmanager.js]: Created a utility to get data object.

File: src/packet.js (hunk 2 above)
Comment (palakrathi) [reply src/packet.js]: Removed.

**Review by ophinjay — APPROVED** [review]

**Pull request merged**